libspawner/src/win32/runner.cpp

```diff
@@ -136,6 +139,27 @@ void runner::restore_original_environment(const runner::env_vars_list_t& origina
     }
 }
 
+void runner::analyze_executable(const std::string& exe_path) {
+    LOADED_IMAGE exe_image;
+    std::unique_ptr<char> exe_path_rw(_strdup(exe_path.c_str()));
```

Comment:
WinAPI. It requests `LPSTR`, which is non-constant modifyable C-string (i.e. `char*`).
`c_str()` returns `const char*`. Passing this into function will cause UB (although MSVC is quietly silent).